turbopack/crates/turbopack-node/src/transforms/postcss.rs

```diff
@@ -250,7 +255,7 @@ impl Source for JsonSource {
                     .append(key.clone())
                     .append(".json".into()),
             )),
-            None => Ok(AssetIdent::from_path(self.path)),
+            None => Ok(AssetIdent::from_path(self.path.append(".json".into()))),
```

Comment:
Why appending `.json`? `self.path` already has `.json`.

Reply:
Ah I see. `postcssrc` doesn't have json. Do we need it?

Reply:
Exactly. I was under the impression that this extension is used for determining the module type (or something to that end)?